README.md

```diff
@@ -1 +1,23 @@
-# tk-framework-desktopclient
\ No newline at end of file
+## Documentation
+This repository is a part of the Shotgun Pipeline Toolkit.
+
+- For more information about this app and for release notes, *see the wiki section*.
+- For general information and documentation, click here: https://support.shotgunsoftware.com/entries/95441257
+- For information about Shotgun in general, click here: http://www.shotgunsoftware.com/toolkit
+
+## Using this app in your Setup
+All the apps that are part of our standard app suite are pushed to our App Store.
+This is where you typically go if you want to install an app into a project you are
+working on. For an overview of all the Apps and Engines in the Toolkit App Store,
+click here: https://support.shotgunsoftware.com/entries/95441247.
+
+## Have a Question?
+Don't hesitate to contact us! You can find us on support@shotgunsoftware.com
+
+## Use the standalone client
+
+This repository can be used as a command line tool. In order to make a request to a Desktop Server, you can execute a the `client.py` file using `python client.py`
```

Comment:
Maybe we could say Shotgun Create and Shotgun Desktop? Desktop Server sounds too technical and it's not clear what we mean by it.